tests/pipes/ss1_output_test_data.json

```diff
@@ -0,0 +1 @@
+["P3029"]
```

Comment:
Just a nitpick, really, Maybe we should move this file to a `mocked_data` folder a - la PR #17 ?